Packs/Cymulate/Integrations/Cymulate_v2/Cymulate_v2.py

```diff
@@ -1273,14 +1283,15 @@ def list_lateral_movement_template_command(client: Client) -> CommandResults:
     return command_results
 
 
-def start_lateral_movement_assessment_command(client: Client, agent_name: str, template_id: str,
+def start_lateral_movement_assessment_command(client: Client, agent_name: str, template_id: str, agent_Profile_Name: str,
```

Comment:
Also here, note the difference between the name of the arg in the yml file and the function

Reply:
changed for just in case